packages/@aws-cdk/app-delivery/README.md

```diff
@@ -52,9 +52,13 @@ const source = new codepipeline.GitHubSourceAction(pipelineStack, 'GitHub', {
   /* ... */
 });
 const project = new codebuild.PipelineProject(pipelineStack, 'CodeBuild', {
-  /* ... */
+  environment: {
+    buildImage: codebuild.LinuxBuildImage.UBUNTU_14_04_NODEJS_10_1_0,
```

Comment:
I wouldn't specify this in the `README.md`, it could be distracting (or looking like it needs to be that).

Reply:
Made a comment instead as an example.